service/test/agama/storage/manager_test.rb

```diff
@@ -378,13 +380,16 @@
     let(:devicegraph) { "staging-plain-partitions.yaml" }
 
     it "copy needed files, installs the bootloader, sets up the snapshots, " \
-       "copy logs, runs the post-installation scripts, and umounts the file systems" do
+       "copy logs, symlink resolv.conf, runs the post-installation scripts, " \
+       "unlink resolv.conf and umounts the file systems" do
       expect(security).to receive(:write)
       expect(copy_files).to receive(:run)
       expect(bootloader_finish).to receive(:write)
       expect(Yast::WFM).to receive(:CallFunction).with("storage_finish", ["Write"])
       expect(Yast::WFM).to receive(:CallFunction).with("snapshots_finish", ["Write"])
+      expect(network).to receive(:link_resolv)
```

Comment:
NP:

Test looks ok, but it is a bit artificial to me. Indeed, you're checking that `link_resolv` methods and `unlink_resolv` methods were called, but it's not testing the real behavior you wanna ensure.

Of course, this tests shouldn't know the implementation details, but I miss  tests somewhere for both mentioned methods.